functionary/ui/templates/partials/environments/environment_management.html

```diff
@@ -0,0 +1,50 @@
+<label class="label" for="users">
+    <i class="fa fa-user"></i>&nbsp;Users:
+    <button class="button is-small has-text-link is-white singletonActive"
+            hx-trigger="click"
+            hx-get="{% url 'ui:create-environment-member' environment_id %}"
+            hx-target="#add-user"
+            type="button">
+        <span class="fa fa-plus" />
+    </button>
+</label>
+
+<div id="add-user">
+    {% if form %}
+        {% include 'partials/environments/environments_user_update.html' with form=form %}
+    {% endif %}
+</div>
+
+<ul id="users">
+    {% for u in users %}
+        <li class="block ml-4 mb-4">
+            <!-- Add in-line form to switch user's role. Pass in id of user. Maybe with a dropdown menu that posts on select?
+            Let user downgrade their own permissions? -->
+            <p>
+                {{ u.user.first_name }} {{ u.user.last_name }} 
+                <span class="has-text-grey-light">({{ u.user.username }})</span>
+                <span>[{{ u.role }}]</span>
+                <span class="has-text-grey-light">{% if u.inherited %}Inherited{% endif %}</span>
+                <button class="button is-small has-text-link is-white singletonActive"
+                    hx-get="{% url 'ui:update-environment-member' environment_id=environment_id user_id=u.user.id %}"
+                    hx-target="#{{ u.user.username }}"
+                ><span class="fa fa-wrench"></span>
+                </button>
+                {% if not u.inherited %}
+                <button class="button is-small has-text-link is-white singletonActive"
+                    hx-delete="{% url 'ui:delete-environment-member' environment_id=environment_id user_id=u.user.id %}"
+                    hx-confirm="Are you sure you want to remove {{ u.user.username }} from this team?"
+                    hx-target="closest li"
+                    hx-swap="outerHTML swap:1s"
+                ><span class="fa fa-trash"></span>
+                </button>
+                {% endif %}
+            </p>
+            <div id="{{ u.user.username }}">
+                {% if form %}
+                    {% include 'partials/teams/environment_update_user.html' with form=form %}
```

Comment:
You may not actually need to pass form here, I think it may get it for free since it's {% include %}'d

Reply:
Fixed. A leftover from before I realized the context gets passed down to included templates